composer.json

```diff
@@ -19,11 +19,11 @@
     },
     "require-dev": {
         "phpspec/phpspec": "^3.4 || ^4.2",
-        "phpspec/prophecy": "^1.8",
         "guzzlehttp/psr7": "^1.4"
     },
     "conflict": {
         "doctrine/instantiator": "<1.0.5",
+        "phpspec/prophecy": "<1.8",
```

Comment:
Does conflict interact with the global require ? Because it's only in conflict for the dev environment, does conflict-dev exist ?

Reply:
Sadly conflict-dev does not exist, so we should keep the old behavior (minimum version in require dev), as the runtime library does not conflict with prophecy < 1.8. It's only the tests.

Reply:
It's possible to simulate conflict-dev by making a dummy package that has exactly one conflict, then require said package in require-dev.

Reply:
good catch. i'd prefer requiring prohecy ^1.8 in require-dev over creating a dummy package.

Reply:
Ouch I didn't think of this issue! Maybe moving to require-dev is good enough?

Reply:
Yes please set thoses packages into the require dev section instead of using conflict